ui/src/org/pentaho/di/ui/core/default.properties

```diff
@@ -447,3 +447,8 @@ lasttype6=N
 lasttype7=N
 lasttype8=N
 lasttype9=N
+min.safari.supported=601
+min.explorer.supported=11
+min.ubuntu.supported=14
+max.ubuntu.supported=16
+pentaho.help.link=https\:\u002F\u002Fhelp.pentaho.com\u002FDocumentation\u002F7.0\u002F0D0\u002F160\u002F000
```

Comment:
these names are not properly simple to understand. 
For some, we test the browser, for others we test the SO. Can we have something like:
min.mac.browser.supported
min.windows.browser.supported
min.ubuntu.os.supported
max.ubuntu.os.supported

pentaho help link too. put a property that connects with the use case. the name sounds to general for the use case we are using